.github/workflows/build-and-test.yml

```
@@ -36,63 +36,133 @@ jobs:
-          pip install -r .binder/requirements.txt
-      - name: Build wheel
-        run: python setup.py sdist bdist_wheel
-      - name: Install from wheel
-        run: pip install --pre --no-index --no-deps --find-links dist/ sktime
-      - name: Run examples
+          python -m pip install .[all_extras,binder]
+      - name: Run example notebooks
         run: build_tools/run_examples.sh
         shell: bash
 
-  build:
+  test-windows:
     needs: code-quality
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-10.15]
+        os: [windows-2019]
         python-version: [3.6, 3.7, 3.8, 3.9]
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Display Python version
-      run: python -c "import sys; print(sys.version)"
-    - name: Install OS packages
-      run: brew install libomp
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r build_tools/requirements.txt
-        pip install pytest-cov
-    - name: Build wheel
-      run: |
-        export CC=/usr/bin/clang
-        export CXX=/usr/bin/clang++
-        export CPPFLAGS="$CPPFLAGS -Xpreprocessor -fopenmp"
-        export CFLAGS="$CFLAGS -I/usr/local/opt/libomp/include"
-        export CXXFLAGS="$CXXFLAGS -I/usr/local/opt/libomp/include"
-        export LDFLAGS="$LDFLAGS -L/usr/local/opt/libomp/lib -lomp"
-        export DYLD_LIBRARY_PATH=/usr/local/opt/libomp/lib
-        export PATH="/usr/local/opt/ccache/libexec:$PATH"
-        python setup.py sdist bdist_wheel
-    - name: Install from wheel
-      run: pip install --pre --no-index --no-deps --find-links dist/ sktime
-    - name: Run tests
-      run: |
-        mkdir -p testdir/
-        cp .coveragerc testdir/
-        cp setup.cfg testdir/
-        cd testdir/
-        pytest --showlocals --durations=10 --pyargs --cov-report=xml --cov=sktime sktime
-    - name: Display coverage report
-      run: ls -l ./testdir/
-    - name: Publish code coverage
-      uses: codecov/codecov-action@v1
-      with:
-        file: ./testdir/coverage.xml
+      - uses: actions/checkout@v2
+
+      - uses: actions/checkout@v2
+      - name: Setup conda
+        uses: s-weigand/setup-conda@v1
+        with:
+          update-conda: true
+          python-version: ${{ matrix.python-version }}
+          conda-channels: anaconda, conda-forge
+      - run: conda --version
+      - run: which python
+
+      - name: Install sktime and dependencies (windows)
+        run: |
+          conda install -c anaconda libpython
+          conda install -c conda-forge "pystan>=2.14,<3.0"
+          conda install -c conda-forge "fbprophet>=0.7.1"
+          conda install -c conda-forge scipy
+          python -m pip install .[all_extras,dev]
+
+      - name: Show dependecies
+        run: python -m pip list
```

Should we use this instead since we're using conda anyway?
```suggestion
        run: conda list
```

---

Sure, this is a diagnostic print so whichever is more convenient